metrics-datadog/src/main/scala/com/ovoenergy/effect/Events.scala

```diff
@@ -0,0 +1,51 @@
+package com.ovoenergy.effect
+
+import com.ovoenergy.effect.Events.Event
+
+/**
+ * Events are a Datadog extension to StatsD
+ * You can also send them over HTTP but since metrics are UDP
+ * we use the UDP approach here too
+ */
+trait Events[F[_]] {
+  def event(event: Event): F[Unit]
+}
+
+object Events {
+
+  sealed abstract class Priority(val value: String)
+
+  object Priority {
+    case object Normal extends Priority("normal")
+    case object Low extends Priority("low")
+  }
+
+  sealed abstract class AlertType(val value: String)
+
+  object AlertType {
```

Comment:
Would be tempted to call this `EventType`,  otherwise `Success` seems a bit weird 😁

Reply:
it is sadly what Datadog call this field :(

Reply:
Oh 😢

Reply:
In that case, carry on!